File: Producto/backend/src/main/java/com/proyecto/macrofit/rutinaejercicio/model/Rutina.java
package com.proyecto.macrofit.rutinaejercicio.model;

import lombok.Data;

@Data
public class Rutina {

    private Integer idRutina;
    private Integer idUsuarioCreador;
    private String nombreRutina;
    private String descripcion;
    private Integer cantidadDias;
    private Boolean esBase;
    private Boolean activoCatalogo;

}

File: Producto/MacroFit/app/src/main/java/com/duoc/macrofit/rutinas/model/Rutina.kt
package com.duoc.macrofit.rutinas.model

import com.google.gson.annotations.SerializedName

data class Rutina(
    val idRutina: Int,

    val idUsuarioCreador: Int?,

    val nombreRutina: String,

    val descripcion: String?,

    val cantidadDias: Int? = 1,

    val esBase: Boolean?,

    val activoCatalogo: Boolean?
)